### ISSUE TITLE
depend on usethis >=1.5.0

### ISSUE THREAD

**Issue by jefferis** [natverse/natmanager#12]
We use ui_code which was added in 1.5.0 according to https://cran.r-project.org/web/packages/usethis/news/news.html

**Comment by jefferis** [comment]
Otherwise if you have an older version of usethis already

```
install.packages("natmanager")
Warning in install.packages :
  cannot open URL 'https://cran.rstudio.com/bin/macosx/el-capitan/contrib/3.5/PACKAGES.rds': HTTP status was '404 Not Found'
  There is a binary version
  available but the source
  version is later:
           binary source
natmanager  0.4.0  0.4.6
           needs_compilation
natmanager             FALSE
installing the source package 'natmanager'
trying URL 'https://cran.rstudio.com/src/contrib/natmanager_0.4.6.tar.gz'
Content type 'application/x-gzip' length 8540 bytes
==================================================
downloaded 8540 bytes
* installing *source* package 'natmanager' ...
** package 'natmanager' successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error : object 'ui_code' is not exported by 'namespace:usethis'
ERROR: lazy loading failed for package 'natmanager'
* removing '/Library/Frameworks/R.framework/Versions/3.5/Resources/library/natmanager'
Warning in install.packages :
  installation of package 'natmanager' had non-zero exit status
The downloaded source packages are in
	'/private/var/folders/f6/xwp8q2nj7wqf_gbxvxr45rdr0000gn/T/Rtmpo8BFgZ/downloaded_packages'
```

**Comment by jefferis** [comment]
Closed by #13
